resource-server/okta-aspnetcore-webapi-example/Properties/launchSettings.json

```diff
@@ -3,10 +3,11 @@
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:8000/",
-      "sslPort": 0
+      "applicationUrl": "http://localhost:5000",
+      "sslPort": 44314
```

Comment:
In https://github.com/okta/samples-aspnetcore/pull/23/files#diff-6f9d3b74cd5f3e0e8e6ff04b0b1abe89R38 it says the https port is 5001.  Does that need to be updated here or is this something different?

Reply:
It's 5001 for dotnet CLI (Kestrel). Unfortunately, we can't use the same range of ports for IIS, that's why for that case we are using 44314.